language/stdlib/modules/validator_config.mvir

```diff
@@ -1,9 +1,14 @@
 module ValidatorConfig {
 
+    // TODO(philiphayes): We should probably enforce a max length for these fields
```

Comment:
Good callout. We're planning to move array from the horror of `bytearray` and have some fixed-length byte types (e.g., bytes32) that should be suitable for this.